sensu_configs/init.d/sensu-service

```diff
@@ -319,13 +319,25 @@ status() {
     return 3
 }
 
+validate_config() {
+    start_daemon $USER $pidfile $exec --validate_config "$options"
```

Comment:
Need to modify the PID file argument so not to interfere with the running process.